Singularity/Singularity/Units/GeneralUnit.cs

```diff
@@ -166,12 +168,14 @@ public void Update(GameTime gametime)
                     {
                         mDone = false;
                         //Care!!! DO NOT UNDER ANY CIRCUMSTANCES USE THIS PLACEHOLDER
-                        IPlatformAction action = new ProduceMineResource(null, null);
-                        mAssignedTask = mDirector.GetDistributionManager.RequestNewTask(this, Job, Optional<IPlatformAction>.Of(action));
+                        // IPlatformAction action = new ProduceMineResource(platform: null, resourceMap: null, director: ref mDirector);
```

Comment:
I needed a placeholder and it was late, I didnt think much about it, to be honest.

Reply:
Alles klar :D